From f7967be4be10edf13c54d490ff57bb83a8b2b6ff Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 3 Oct 2018 21:12:53 +0200
Subject: [PATCH 1/1] efi_loader: EFI_ENTRY print device path

If parameters are of type struct efi_device_path *, use %pD to print
the parameter in EFI_ENTRY.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_boottime.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/efi_loader/efi_boottime.c b/lib/efi_loader/efi_boottime.c
index a9f79969a3..4e13ee1b21 100644
--- a/lib/efi_loader/efi_boottime.c
+++ b/lib/efi_loader/efi_boottime.c
@@ -2342,7 +2342,7 @@ static efi_status_t EFIAPI efi_locate_device_path(
 	u8 *remainder;
 	efi_status_t ret;
 
-	EFI_ENTRY("%pUl, %p, %p", protocol, device_path, device);
+	EFI_ENTRY("%pUl, %pD, %p", protocol, device_path, device);
 
 	if (!protocol || !device_path || !*device_path || !device) {
 		ret = EFI_INVALID_PARAMETER;
@@ -2894,7 +2894,7 @@ static efi_status_t EFIAPI efi_connect_controller(
 	efi_status_t ret = EFI_NOT_FOUND;
 	struct efi_object *efiobj;
 
-	EFI_ENTRY("%p, %p, %p, %d", controller_handle, driver_image_handle,
+	EFI_ENTRY("%p, %p, %pD, %d", controller_handle, driver_image_handle,
 		  remain_device_path, recursive);
 
 	efiobj = efi_search_obj(controller_handle);
-- 
2.19.0

